### ISSUE TITLE
[FEATURE] Integrate with or link to Flow playground

### ISSUE THREAD

**Issue by bartolomej** [muttoni/contract-browser#10]
### Issue to be solved

I really like how crates.io links to the Rust playground where you can try and play around with the package you are looking at.

![Screenshot 2024-06-04 at 22 05 45](https://github.com/muttoni/contract-browser/assets/36109955/e6bb833e-4785-4873-ba70-1e63e686b1d7)

I believe this would be useful for developers exploring new Cadence packages/contracts (as the number of contracts grows and the composability becomes even more relevant) + it is a good step towards better integration/connection with other tools in the Flow ecosystem.

### Suggest A Solution

Right now, this is not possible to achieve with the current playground https://play.flow.com. But we are working on a new web-based playground (see https://github.com/onflow/developer-grants/issues/260) that could potentially recreate a very similar environment to the local one, where the user could:
- install the specified contract (e.g. the one that he viewed previously on contractbrowser.com) using the standard package manager tool - this could be done automatically as the page is visited by providing a URL parameter on the contractbrowser.com side
- view and play around with the standard transactions/scripts that work with the specified contract (by leveraging FLIX)

**Comment by bartolomej** [comment]
This is also sort of related to https://github.com/muttoni/contract-browser/issues/9. It's still a question if the interaction and playground tool should be available within the same app (basically be one tool that supports both emulated and testnet/mainnet mode) or not.

**Comment by muttoni** [comment]
So you're suggesting to link out to the Flowser playgroud? Sure! What is the ETA?

**Comment by bartolomej** [comment]
> So you're suggesting to link out to the Flowser playgroud? Sure! What is the ETA?

Yes! The project is still in its early development stage, but we may have a working MVP somewhat soon (~ 1-2 months), so we may do a beta release at that point.

FYI some of the development work can be tracked here: https://github.com/onflowser/flow-wasm
